models/issue_tracked_time.go

```diff
@@ -24,20 +24,49 @@ type TrackedTime struct {
 	Time        int64     `json:"time"`
 }
 
+// TrackedTimeList is a List ful of TrackedTime's
+type TrackedTimeList []*TrackedTime
+
 // AfterLoad is invoked from XORM after setting the values of all fields of this object.
 func (t *TrackedTime) AfterLoad() {
 	t.Created = time.Unix(t.CreatedUnix, 0).In(setting.DefaultUILocation)
 }
 
 // APIFormat converts TrackedTime to API format
 func (t *TrackedTime) APIFormat() *api.TrackedTime {
+	user, err := GetUserByID(t.UserID)
```

Comment:
Because you're now querying the db in the API format you need to check that this never happens within a session and if it does then you need to make an `apiFormat(e Engine)` variant which would run in that session.

Reply:
add *User and *Issue to normal strukt so APIFormat() has no DB query anymore